.github/workflows/ci.yml

```
@@ -0,0 +1,73 @@
+        image: mysql:8
+        env:
+          MYSQL_ROOT_PASSWORD: rootpassword
+          MYSQL_DATABASE: laravel_test
+          MYSQL_USER: laravel_user
+          MYSQL_PASSWORD: laravel_password
+        ports:
+          - 3306:3306
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)